test/tester.c

```diff
@@ -3428,6 +3428,36 @@ START_TEST(test_local_disk_rpm_get)
 }
 END_TEST
 
+START_TEST(test_local_disk_link_type)
+{
+    int rc = 0;
+    lsm_disk_link_type link_type = LSM_DISK_LINK_TYPE_UNKNOWN;
+    lsm_error *lsm_err = NULL;
+
+    rc = lsm_local_disk_link_type_get("/dev/sda", &link_type,  &lsm_err);
+    if (rc != LSM_ERR_OK)
+        lsm_error_free(lsm_err);
```

Comment:
Since we cannot tell whether '/dev/sda' exists or whether current user has sufficient permission, we
could never know what kind of error we could get.
But we should not get LIB_BUG for sure. Will update test case to check LIB_BUG.